pkg/localConfigProvider/mock_localConfigProvider.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by MockGen. DO NOT EDIT.
 // Source: localConfigProvider.go
 
-// Package localConfigProvider is a generated GoMock package.
+// Package mock_localConfigProvider is a generated GoMock package.
```

Comment:
The package is `localConfigProvider`

Reply:
Updated tho i dont get why it is still showing this in diff

Reply:
Guess it's gone now from the diff